.github/workflows/integration-azure.yaml

```diff
@@ -50,6 +50,10 @@ jobs:
         run: cat .env
       - name: Build test app
         run: make docker-build
+      - name: Prepare Git SSH secret
+        run: echo $GIT_SSH_SECRET > git-ssh-secret.yaml
+        env:
+          GIT_SSH_SECRET: ${{ secrets.GIT_SSH_SECRET }}
```

Comment:
@stefanprodan Please add this secret to the repo, it's in the file `git-ssh-key.yaml` of the item `Matheus Azure Int Tests` in the fluxcd 1password (same we used for Azure int tests)

Reply:
Done

Reply:
Storing a whole YAML file in a secret is bad practice, GitHub could show the secret in logs due to formatting changes.